src/test/src/StringUtilityTest.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iomanip>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+#include "StringUtility.h"
+
+namespace {
```

Comment:
The test should be outside the anonymous namespace because it needs to be referenced by main().

Reply:
done